main.js

```diff
@@ -0,0 +1,11 @@
+
+  $.ajax({
+    url: "https://randomuser.me/api/?inc=name,email,id&results=10",
+    dataType: "json",
+    success: function (data) {   
```

Comment:
it is better to filter your data before sending it to render. You do not need to send everything that you received from the API to the handlebars. Make an object with only the data you will be rendering.